test/common/alerts_exist.go

```diff
@@ -396,6 +396,12 @@ var expectedRules = []alertsTestRule{
 			"RHMIInstallationControllerStoppedReconciling",
 		},
 	},
+	{
+		File: "redhat-rhmi-middleware-monitoring-operator-install-upgrade-alerts.yaml",
```

Comment:
How about removing the hard coded Namespace prefix? The same way it's done for other alerts, e.g: `File: NamespacePrefix + "middleware-monitoring-operator-install-upgrade-alerts.yaml",`

Reply:
Agreed. There is another constant, that suits the need here better: ```	NamespacePrefix                   = "redhat-rhmi-"
	RHMIOperatorNamespace             = NamespacePrefix + "operator"
	MonitoringOperatorNamespace       = NamespacePrefix + "middleware-monitoring-operator"``` . Changed it to MonitoringOperatorNamespace +